drivers/util.py

```diff
@@ -1228,8 +1228,10 @@ def daemon():
 #########################
 
 if __debug__:
+    # FIXME this needs some explanation as to why os.environ.get() is
+    # not called unconditionally
     try:
-        XE_IOFI_IORETRY
+        XE_IOFI_IORETRY         # pylint: disable=used-before-assignment
```

Comment:
This whole thing looks truly weird, it was weird before so it's not a reflection of this change but I wonder whether we might not be better seeing if this is actually even used and removing it if not.

Reply:
This does indeed seem to be entirely unused and legacy cruft so please just delete the entire `if __debug__:` block